packages/manager/modules/pci/src/projects/project/notebooks/add/components/notebook-privacy-settings/notebook-privacy-settings.html

```diff
@@ -0,0 +1,34 @@
+<div>
+    <!--Notebook Privacy Networks -->
+    <div class="row">
+
+        <!--Settings restricted-->
+        <oui-select-picker
```

Comment:
You can try to use a ng-repeat...

Reply:
I can't, the both items they are completely separate (not in the same array).
And we can't have others models.